.packit.yaml

```diff
@@ -0,0 +1,24 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: osbs-client.spec
+
+# add or remove files that should be synced
+synced_files:
```

Comment:
Please check https://github.com/containerbuildsystem/dockerfile-parse/pull/95/files

It seems that many keys here are not needed anymore.

Reply:
I removed all unnecessary parts. Kept only part applicable for PR(s).

Reply:
https://github.com/StykMartin/osbs-client/pull/2 <-- You can see how it looks like here. 
Packit conf https://github.com/StykMartin/osbs-client/commit/03ee3700b8cc9e67560cfb3c6e49aebb5630ff9d